src/animators/index.ts

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
What are the big pros/cons of re-exporting in index vs. consumers doing

`import * as BaseAnimator from "./animators/baseAnimator"`